EventFilter/GEMRawToDigi/plugins/GEMRawToDigiModule.cc

```diff
@@ -71,6 +72,12 @@ GEMRawToDigiModule::GEMRawToDigiModule(const edm::ParameterSet& pset)
   if (useDBEMap_) {
     gemEMapToken_ = esConsumes<GEMeMap, GEMeMapRcd, edm::Transition::BeginRun>();
   }
+  if (ge21Off_ && fedIdStart_ == FEDNumbering::MINGEMFEDID && fedIdEnd_ == FEDNumbering::MAXGEMFEDID) {
+    fedIdEnd_ = FEDNumbering::MINGE21FEDID - 1;
```

Comment:
I wonder what is the advantage of having this `ge21Off_` switch: wouldn't it be the same setting appropriately the "fedIdStart" and "fedIdEnd" parameters in the configuration, and thus get rid of this extra redundant bool parameter?

Reply:
@perrotta That is what I did originally, but the suggestion was to use the bool because otherwise we need to copy the constants from FEDNumbering into the python configuration. So, the tradeoff was adding the switch, and checking the user isn't trying to set the FED IDs themselves, or copying the numbers into the python config and having a redundant copy of the constants copy and pasted from the c++ header (I believe there is no way to access these enum values by name in the python?).

Reply:
Ah, ok, I see... Thank you @watson-ij